pages/[username]/[collectionId]/index.tsx

```diff
@@ -28,9 +30,35 @@ export default function CollectionGallery({ collectionId, username }: Collection
 
 export const getServerSideProps: GetServerSideProps<CollectionGalleryProps> = async ({
   params,
-}) => ({
-  props: {
-    username: params?.username ? (params.username as string) : undefined,
-    collectionId: params?.collectionId ? (params.collectionId as string) : undefined,
-  },
-});
+}) => {
+  const username = params?.username ? (params.username as string) : undefined;
+  const collectionId = params?.collectionId ? (params.collectionId as string) : undefined;
+  return {
+    props: {
+      username,
+      collectionId,
+      metaTags: collectionId
+        ? [
+            // TODO: fetch collection for better title?
```

Comment:
are these todos for this PR or the future? collection name would be cool for sure

Reply:
> are these todos for this PR or the future? collection name would be cool for sure

Mostly for future! The reason I can't do this right now is we're inside `getServerSideProps`, which doesn't do any backend queries yet. We'd need to query for the collection name if we wanted to use that in the title.

Reply:
sounds gooddd